tss-esapi/src/error/return_code/tpm/format_zero/error.rs

```diff
@@ -50,40 +58,40 @@ impl std::error::Error for TpmFormatZeroErrorResponseCode {}
+            TpmFormatZeroError::Pcr => write!(f, "PCR check fail."),
+            TpmFormatZeroError::PcrChanged => write!(f, "PCR have changed since checked."),
+            TpmFormatZeroError::Upgrade => write!(f, "For all commands other than TPM2_FieldUpgradeData(), this code indicates that the TPM is in field upgrade mode; for TPM2_FieldUpgradeData(), this code indicates that the TPM is not in field upgrade mode."),
+            TpmFormatZeroError::TooManyContexts => write!(f, "Context ID counter is at maximum."),
+            TpmFormatZeroError::AuthUnavailable => write!(f, "'authValue' or 'authPolicy' is not available for selected entity."),
+            TpmFormatZeroError::Reboot => write!(f, "A _TPM_Init and Startup(CLEAR) is required before the TPM can resume operation."),
+            TpmFormatZeroError::Unbalanced => write!(f, "The protection algorithms (hash and symmetric) are not reasonably balanced. The digest size of the hash must be larger than the key size of the symmetric algorithm."),
+            TpmFormatZeroError::CommandSize => write!(f, "Command 'commandSize' value is inconsistent with contents of the command buffer; either the size is not the same as the octets loaded by the hardware interface layer or the value is not large enough to hold a command header."),
+            TpmFormatZeroError::CommandCode => write!(f, "Command code not supported."),
+            TpmFormatZeroError::AuthSize => write!(f, "The value of 'authorizationSize' is out of range or the number of octets in the authorization area is greater than required."),
+            TpmFormatZeroError::AuthContext => write!(f, "Use of an authorization session with a context command or another command that cannot have an authorization session."),
+            TpmFormatZeroError::NvRange => write!(f, "NV offset+size is out of range."),
+            TpmFormatZeroError::NvSize => write!(f, "Requested allocation size is larger than allowed."),
+            TpmFormatZeroError::NvLocked => write!(f, "NV access locked."),
+            TpmFormatZeroError::NvAuthorization => write!(f, "NV access authorization fails in command actions."),
+            TpmFormatZeroError::NvUninitialized => write!(f, "An NV Index is used before being initialized or the state saved by TPM2_Shutdown(STATE) could not be restored."),
+            TpmFormatZeroError::NvSpace => write!(f, "Insufficient space for NV allocation."),
+            TpmFormatZeroError::NvDefined => write!(f, "NV Index or persistent object already defined."),
+            TpmFormatZeroError::BadContext => write!(f, "Context in TPM2_ContextLoad() is not valid."),
+            TpmFormatZeroError::CpHash => write!(f, "'cpHash' value already set or not correct for use."),
+            TpmFormatZeroError::Parent => write!(f, "Handle for parent is not a valid parent."),
+            TpmFormatZeroError::NeedsTest => write!(f, "Some function needs testing."),
```

Comment:
This looks a little bit off. For me even `Function needs testing` is OK.

Reply:
Fixed.